src/scripts/main.js

```diff
@@ -2,7 +2,18 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tableBody = document.querySelector('.dashboard tbody');
 
-// write your code here
+for (const person of people) {
+  const row = document.createElement('tr');
+
+  row.innerHTML = `
+  <td>${person.name}</td>
+  <td>${person.sex === 'f' ? 'Female' : 'Male'}</td>
+  <td>${person.born}</td>
+  <td>${person.died}</td>
+  <td>${person.died - person.born}</td>
+  <td>${Math.ceil(person.died / 100)}</td>`;
```

Comment:
```suggestion
  row.innerHTML = `
    <td>${person.name}</td>
    <td>${person.sex === 'f' ? 'Female' : 'Male'}</td>
    <td>${person.born}</td>
    <td>${person.died}</td>
    <td>${person.died - person.born}</td>
    <td>${Math.ceil(person.died / 100)}</td>
  `;
```
Вау ну майже реакт:)